src/components/analyser/search/SearchBar.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { TextField, InputAdornment } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import Search from '@material-ui/icons/Search';
+import * as API from '../../../analyser/APIClient';
+import SearchOption from './SearchOption';
+import { ErrorCode } from '../../../Errors';
+import ErrorMessage from '../ErrorMessage';
+
+export type SearchBarProps = {
+  token: string;
+};
+
+const SearchBar: React.FC<SearchBarProps> = ({ token }) => {
+  const [stocks, setStocks] = React.useState<API.Stock[]>();
+  const [error, setError] = React.useState<ErrorCode | undefined>();
+
+  const isMounted = React.useRef(true);
+  const fetch = async () => {
+    setError(undefined);
+    try {
+      const s = await API.listStocks(token);
+      if (isMounted.current) {
+        setStocks(s);
+      }
+    } catch (e) {
+      if (isMounted.current) {
+        setError(e.message);
+      }
+    }
+  };
+
+  React.useEffect(() => {
+    fetch();
+    return () => {
+      isMounted.current = false;
+    };
+    // deps must be empty because the function should only be called on mount.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div style={{ width: 280 }}>
+      {error && (
+        <ErrorMessage
+          error={error}
+          messageKey="analyser.dashboard.errorMessage"
+        />
+      )}
+      {stocks && (
+        <Autocomplete
+          id="search"
+          freeSolo
+          options={stocks}
+          getOptionLabel={(option) =>
+            `${option.symbol} ${option.name}${option.ISIN}${option.WKN}`
+          }
+          renderOption={(option) => <SearchOption stock={option} />}
+          renderInput={(params) => (
+            <TextField
```

Comment:
we need styling, where is it at?

- placeholder text can't be shown correctly
- searchbar is fixed to the right corner of the header
- not mobile friendly
- search bar is too small

<img width="562" alt="image" src="https://user-images.githubusercontent.com/29731862/113296283-8b820c80-92f9-11eb-975e-716e4df12a5c.png">
<img width="1751" alt="image" src="https://user-images.githubusercontent.com/29731862/113296305-95a40b00-92f9-11eb-9cc0-cdbc57d84b0d.png">